worker/worker.go

```diff
@@ -517,6 +519,57 @@ func (w *worker) rhpFormHandler(jc jape.Context) {
 	})
 }
 
+func (w *worker) rhpBroadcastHandler(jc jape.Context) {
+	var fcid types.FileContractID
+	if jc.DecodeParam("id", &fcid) != nil {
+		return
+	}
+
+	// Fetch contract from bus.
+	ctx := jc.Request.Context()
+	c, err := w.bus.Contract(ctx, fcid)
+	if jc.Check("could not get contract", err) != nil {
+		return
+	}
+	gp, err := w.bus.GougingParams(ctx)
+	if jc.Check("could not get gouging parameters", err) != nil {
+		return
+	}
+	rk := w.deriveRenterKey(c.HostKey)
+	ctx = WithGougingChecker(ctx, w.bus, gp)
```

Comment:
Do you need the gouging checker on the `ctx` here? Seems like you are not using it